manifest_staging/charts/secrets-store-sync-controller/values.yaml

```
@@ -26,7 +25,7 @@ validatingAdmissionPolicies:
     - "kubernetes.io/service-account-token"
 
 image:
-  repository: secrets-sync-controller # e.g. my-registry.example.com/my-repo
+  repository: secrets-store-sync-controller # e.g. my-registry.example.com/my-repo
   pullPolicy: IfNotPresent
   tag: "v0.0.1 "
```

Is there a reason for a gap here at the end?
```suggestion
  tag: "v0.0.1"
```